lib/logstash/inputs/dead_letter_queue.rb

```diff
@@ -8,27 +8,37 @@ class LogStash::Inputs::DeadLetterQueue < LogStash::Inputs::Base
 
   default :codec, 'plain'
 
+  # Path to the dead letter queue directory which was created by a Logstash instance.
+  # This is the path from where "dead" events are read from.
   config :path, :validate => :path, :required => true
```

Comment:
we should probably note that this would be the exact same value of the LS setting of `path.dead_letter_queue`, which defaults to `{path.data}/dead_letter_queue`